Fixes airlock/door placements in away sites

Ryan180602 opened the pull request:
:cl: SDTheCyanWyan
maptweak: Replaced away site two-door mapping with double doors and double airlocks with singles.
/:cl:

Some rooms in Derelict Station, Errant Pisces, Lar Maria, Mining Asteroid and Mining Signal, usually had entrances using two single  doors instead of multi-tiled doors, causing misalignment upon generation. 
Airlocks with two internal and external doors have been trimmed down to one each.

SDTheCyanWyan commented:
Was recommended to fix the tiling under some airlocks on away sites as well. Will search and fix those tomorrow.

SierraKomodo commented:
> Was recommended to fix the tiling under some airlocks on away sites as well. Will search and fix those tomorrow.

Probably unnecessary as there's a separate PR up to handle that now by the looks of it - https://github.com/Baystation12/Baystation12/pull/30012

SierraKomodo reviewed (dismissed):
While it's not as vital for away sites, pipes under walls should generally be avoided as a practice when possible, instead routing them under doors or windows - Some of your door/wall changes result in pipes under walls.

Ryan180602 committed:
Fixes airlock/door placements in away sites

Verticle double door/airlock door placements usually mean they misalign. No more.

SDTheCyanWyan commented:
Made the proposed changes, rebased as well. Put up an issue to hopefully have the cause be corrected, so the PR can be dropped if need be; but as discussed, double single-tiled doors and airlock doors serve really no use, when they can be replaced with multi-tiled doors, and for atmospheric airlocks, with one internal and external airlock doors.

Spookerton reviewed (approved):
Eh. We'll see what happens.
